integration-test/875-camp-grounds-zoom.py

```diff
@@ -44,6 +44,7 @@
 
 # small campground in point zoom 13
 # http://www.openstreetmap.org/way/417405356
+# min zoom matches in landuse layer, but not in pois for 13
```

Comment:
Is there anything left for us here? If not, take out the comment to reduce confusion.